package.json

```diff
@@ -2,68 +2,81 @@
   "name": "@coderscamp2020/project-javascript-typescript-starwarsquiz",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "source": "index.html",
+  "browserslist": [
+    "> 0.5%, last 2 versions, not dead"
+  ],
   "scripts": {
-    "clean": "rimraf .cache && rimraf coverage && rimraf dist && rimraf node_modules",
-    "prebuild": "rimraf dist",
-    "build": "parcel build index.html --public-url ./",
-    "format": "prettier --write \"src/**/*.js\" \"src/**/*.ts\" \"test/**/*.js\" \"test/**/*.ts\"",
-    "start:dev": "parcel -p 8765 watch index.html",
-    "test": "jest",
-    "test:watch": "jest --watch",
-    "test:cov": "jest --coverage"
+    "clean": "",
+    "prebuild": "",
+    "start": "parcel",
+    "build": "parcel build",
+    "format": "",
+    "start:dev": "",
+    "test": "",
+    "test:watch": "",
+    "test:cov": ""
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/CodersCamp2020/CodersCamp2020.Project.JavaScript.StarWarsQuiz.git"
   },
   "keywords": [],
-  "author": "CodersCamp2020",
+  "author": {
+    "name": "CodersCamp2021",
+    "email": "kontakt@coderscrew.pl",
```

Comment:
Perhaps an email of one of the team members?

Reply:
I will add an email to our Development Manager